process.py

```diff
@@ -81,7 +81,7 @@ def system_profile(hostname, cpu_info, virt_what, meminfo, ip_addr, dmidecode,
         profile['infrastructure_vendor'] = virt_what.generic
 
     if installed_rpms:
-        profile['installed_packages'] = sorted([str(p[0]) for p in installed_rpms.packages.values()])
+        profile['installed_packages'] = sorted([p[0].nevra() for p in installed_rpms.packages.values()])
```

Comment:
Did some fiddling and I think all we need to do to fix the bug is this
`sorted([p[0].nevra for p ...`
So don't actually call `nevra()` but just use the property.

Reply:
Thanks, Fixed